Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Delete.ts

```diff
@@ -27,43 +31,32 @@ export async function deleteDocumentByDocumentUuid(
   let meadowlarkId: MeadowlarkId = '' as MeadowlarkId;
 
   try {
-    await client.query('BEGIN');
+    await beginTransaction(client);
     // Find the alias meadowlarkIds for the document to be deleted
-    const documentAliasIdsResult: QueryResult = await client.query(
-      findAliasMeadowlarkIdsForDocumentByDocumentUuidSql(documentUuid),
+    const documentAliasIdsResult: MeadowlarkAlias[] = await findAliasMeadowlarkIdsForDocumentByDocumentUuid(
+      client,
+      documentUuid,
     );
     // Each row contains documentUuid and corresponding meadowlarkId (meadowlark_id),
     // we just need the first row to return the meadowlark_id
     meadowlarkId =
-      documentAliasIdsResult?.rowCount != null && documentAliasIdsResult.rowCount > 0
-        ? documentAliasIdsResult.rows[0].meadowlark_id
-        : ('' as MeadowlarkId);
+      (documentAliasIdsResult?.length ?? 0) > 0 ? documentAliasIdsResult[0].meadowlark_id : ('' as MeadowlarkId);
```

Comment:
You can simplify `(documentAliasIdsResult?.length ?? 0) > 0` down to `documentAliasIdsResult.length > 0` when you are outside SqlHelper. You are already promising it will return an array in the type signature, and you have control over that.